docs/src/main/sphinx/release/release-356.md

```diff
@@ -0,0 +1,73 @@
+# Release 356 (XX Apr 2021)
+
+## General
+
+* Add support for {doc}`/sql/match-recognize`. ({issue}`6111`)
+* Add {func}`soundex` function. ({issue}`4022`)  
+* Introduce `system.metadata.materialized_view_properties` table for listing available materialized view properties. ({issue}`7615`)
+* Introduce planning time limit which can be specified using `query.max-planning-time` config property ({issue}`7213`)
+* Improve cancellation of queries during planning phase ({issue}`7213`)
+* Improve performance of `ORDER BY ... LIMIT` queries over a `LEFT JOIN`. ({issue}`7028`)
+* Improve performance of queries with predicates on boolean columns ({issue}`7263`)
+* Improve planning time for queries with large `IN` predicates. ({issue}`7556`)
+* Improve performance of queries that contain join on `VARCHAR` keys of different length. ({issue}`7644`)
+* Reduce coordinator network overhead when scheduling queries. ({issue}`7351`)
+* Fix join spill deadlock. ({issue}`7455`)
```

Comment:
nit: Maybe `Fix possible deadlock for JOIN queries when spilling is enabled`? `Fix possible deadlock for JOIN queries which spill to disk`?